src/main.cpp

```diff
@@ -741,6 +741,9 @@ QString UsageArguments ( char **argv )
         "  -j, --nojackconnect   disable auto Jack connections\n"
         "  --ctrlmidich          MIDI controller channel to listen\n"
         "  --clientname          client name (window title and jack client name)\n"
+#ifndef HEADLESS
```

Comment:
I actually don't want to this to be shown in the official command line options. It is just a "pro tool" for developers which want to optimize internal algorithm parameters. For "normal" users this would just confuse them. It is what it it: a hidden development feature.

Reply:
OK. I'm fine with that. Should I drop this chunk then?

Reply:
Yes, please.